.eslintrc

```diff
@@ -45,6 +45,39 @@
 			{
 				"required": "id"
 			}
-		]
+		],
+		"jsdoc/check-tag-names": [
+			"error",
+			{
+				"definedTags": [
+					"format"
+				]
+			}
+		],
+		"yoda": [
+			"error",
+			"never"
+		],
```

Comment:
I assume the old value were `[ "always", "error" ]` and this is to stay aligned with WP coding standard. I don't get why it should be change to `"never"`.

Reply:
I tried both always/never variants =) Getting violations in both cases, but less with 'never'. Migrating to 'always', thanks for the feedback!